docs/root/configuration/upstream/cluster_manager/cluster_stats.rst

```diff
@@ -69,7 +69,9 @@ Every cluster has a statistics tree rooted at *cluster.<name>.* with the followi
   upstream_rq_cancelled, Counter, Total requests cancelled before obtaining a connection pool connection
   upstream_rq_maintenance_mode, Counter, Total requests that resulted in an immediate 503 due to :ref:`maintenance mode<config_http_filters_router_runtime_maintenance_mode>`
   upstream_rq_timeout, Counter, Total requests that timed out waiting for a response
+  upstream_rq_timeout_budget_percent_used, Histogram, What percentage of the global timeout was used waiting for a response
```

Comment:
Perhaps make a new section for these 2 stats and describe the timeout precision concept.